test/common/router/config_impl_test.cc

```diff
@@ -3213,42 +3074,32 @@ name: foo
     }
   };
 
-  TestConfigImpl v1_json_config(parseRouteConfigurationFromJson(v1_json), factory_context_, true);
-  testConfig(v1_json_config);
-
   TestConfigImpl v2_yaml_config(parseRouteConfigurationFromV2Yaml(v2_yaml), factory_context_, true);
-  testConfig(v2_yaml_config, true);
+  testConfig(v2_yaml_config);
```

Comment:
Since this is only invoked once now, maybe inline the lambda?

Reply:
sure